lib/Dialect/TTIR/Transforms/EraseInverseOps/ReduceCommutePatterns.cpp

```diff
@@ -0,0 +1,191 @@
+// SPDX-FileCopyrightText: (c) 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ttmlir/Dialect/TTIR/Transforms/EraseInverseOps/EraseInverseOps.h"
+
+#include "mlir/IR/BuiltinTypes.h"
+#include "ttmlir/Dialect/TTCore/IR/Utils.h"
+#include "ttmlir/Dialect/TTIR/IR/TTIROps.h"
+#include "ttmlir/Dialect/TTIR/Utils/Utils.h"
+#include <flatbuffers/array.h>
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/Support/raw_ostream.h"
+#include "mlir/IR/BuiltinAttributes.h"
```

Comment:
:warning: **llvm\-include\-order** :warning:
\#includes are not sorted properly
```suggestion
#include <flatbuffers/array.h>
```